config/fxdata/magic.cfg

```diff
@@ -18,27 +18,66 @@
+;   FEAR - Causes the target to become momentarily scared, causing it to flee and avoid combat. 'SPELL_FEAR' by default.
+;   HEAL - Indicates that the spell heals the target. If linked, level-based 'Power' values are used for the amount of health restored. 'SPELL_HEAL' by default.
+;   TELEPORT - If self-cast, jumps to its next objective or place to fulfill its needs. If cast on a target, jumps it to its lair or flee point. 'SPELL_TELEPORT' by default.
+;   TIMEBOMB - Sets a countdown on the target, making it move toward enemies to explode on impact, dealing damage from the linked shot. 'SPELL_TIME_BOMB' by default.
+;   WIND - Not functional on spells, exists only for immunity.
 SpellFlags = 0
 ; The creature it summons, the experience level and the amount of them. Negative experience levels/amounts are relative to summoning creature.
 SummonCreature = 0 0 0
-; Duration of the spell, provided the duration does not come from the linked power.
+; Sets the duration of the CHICKEN transformation transition.
+Countdown = 0
+; The spell’s duration depends on its link to a power. If linked, it prioritizes the linked power’s duration field, then its power field.
+; If the 'Duration' field is set to 0, the level-based 'Power' values of the linked power are used. If not linked, the spell’s own duration is used.
+; For example, 'SPELL_HEAL' is linked to 'POWER_HEAL_CREATURE' with 'Duration = 16', so its duration is always 16, regardless of the creature’s level.
 Duration = 0
+; Sets the health amount to recover when HEAL is used without being linked to a power.
+HealingRecovery = 0
+; Amount of damage the spell inflicts on the target per tick of frequency.
+Damage = 0
+; Frequency at which the damage is applied, in game turns. Set to 0 for damage to apply only once.
+DamageFrequency = 0
+; Type of damage inflicted. If set to 'Restoration', it heals the target instead.
+DamageType = None
 ; Effect or 'effectElement' (negative numbers) surround the target for the duration of the spell. Accepts both names and numbers.
 AuraEffect = NULL
-; Self casted spells causes their effect on the caster.
-; The first argument is 1 if self casting is enabled, 0 otherwise.
-; The 2nd argument provides a sound sample index to be played when the spell is self-casted.
+; The duration for which the aura effect is active.
+AuraDuration = 0
+; The frequency at which the aura effect is applied. Set to 0 for the aura effect to apply a single time.
+AuraFrequency = 0
+; Determines whether the spell can be used with the script command or if its sound sample index is played.
+; The first argument is 1 if enabled, 0 if disabled. The second argument specifies the sound sample index for self-casting.
 SelfCasted = 0 0
-; The interface icons.
+; Sprite with big size and medium size icon of the spell.
 SymbolSprites = 0 0
+; Spell can have special properties to modify its behaviour:
+;   CLEANSE - Spell will clear the assigned SpellFlags instead of applying them.
```

Comment:
I would say, cleanse first and without a duration, then apply the new spell effect second which does have a duration. But it would be a weird way to configure the spell.